gradle/dependencies.gradle

```diff
@@ -39,9 +39,9 @@ versions += [
   powermock: "1.6.4",
   reflections: "0.9.10",
   rocksDB: "4.11.2",
-  scalaTest: "2.2.6",
+  scalaTest: "3.0.0",
```

Comment:
3.0.1 has been released since the PR was created, so we can probably go straight to that.